Configuration/AlCa/python/autoAlca.py

```diff
@@ -1,30 +1,30 @@
-                  'MinimumBias'    : 'SiStripCalMinBias+TkAlMinBias',
-                  'MuOnia'         : 'TkAlJpsiMuMu+TkAlUpsilonMuMu',
-                  'MuOniaParked'   : 'TkAlJpsiMuMu+TkAlUpsilonMuMu',
-                  'SingleElectron' : 'EcalCalWElectron+EcalUncalWElectron+EcalCalZElectron+EcalUncalZElectron+EcalESAlign+HcalCalIterativePhiSym+HcalCalIsoTrkFilter',
-                  'SingleMu'       : 'MuAlCalIsolatedMu+MuAlOverlaps+TkAlMuonIsolated+DtCalib+MuAlZMuMu+HcalCalHO',
-                  'SingleMuon'     : 'TkAlMuonIsolated+DtCalib+MuAlCalIsolatedMu+MuAlOverlaps+MuAlZMuMu+HcalCalIterativePhiSym',
-                  'SinglePhoton'   : 'HcalCalGammaJet',
-                  'ZeroBias'       : 'SiStripCalZeroBias+TkAlMinBias+LumiPixelsMinBias+SiStripCalMinBias+EcalTrg', 
-                  'StreamExpress'  : 'SiStripCalZeroBias+TkAlMinBias+SiStripPCLHistos+SiStripCalMinBias+SiStripCalMinBiasAfterAbortGap+DtCalib+Hotline+LumiPixelsMinBias',
-                  'StreamExpressHI': 'SiStripCalZeroBias+TkAlMinBiasHI+SiStripPCLHistos+SiStripCalMinBias+SiStripCalMinBiasAfterAbortGap+DtCalibHI',
-                  'ExpressCosmics' : 'SiStripCalZeroBias+TkAlCosmics0T',
+AlCaRecoMatrix = {"AlCaLumiPixels" : "LumiPixels",
+                  "Charmonium"     : "TkAlJpsiMuMu",
+                  "Commissioning"  : "HcalCalIsoTrk",
+                  "Cosmics"        : "TkAlCosmics0T+MuAlGlobalCosmics+HcalCalHOCosmics+DtCalibCosmics",
+                  "DoubleEG"       : "EcalCalZElectron+EcalUncalZElectron+HcalCalIterativePhiSym+HcalCalIsoTrkFilter",
+                  "DoubleElectron" : "EcalCalZElectron+EcalUncalZElectron+HcalCalIsoTrkFilter",
+                  "DoubleMu"       : "MuAlCalIsolatedMu+MuAlOverlaps+DtCalib+TkAlZMuMu",
+                  "DoubleMuon"     : "TkAlZMuMu+MuAlCalIsolatedMu+MuAlOverlaps+MuAlZMuMu+DtCalib",
+                  "DoubleMuParked" : "MuAlCalIsolatedMu+MuAlOverlaps+DtCalib+TkAlZMuMu",
+                  "HLTPhysics"     : "SiStripCalMinBias+TkAlMinBias+HcalCalIsoTrkFilter",
+                  "JetHT"          : "HcalCalDijets+HcalCalIsoTrkFilter+HcalCalIsolatedBunchFilter",
+                  "MET"            : "HcalCalNoise",
+                  "MinimumBias"    : "SiStripCalMinBias+TkAlMinBias",
+                  "MuOnia"         : "TkAlJpsiMuMu+TkAlUpsilonMuMu",
+                  "MuOniaParked"   : "TkAlJpsiMuMu+TkAlUpsilonMuMu",
+                  "SingleElectron" : "EcalCalWElectron+EcalUncalWElectron+EcalCalZElectron+EcalUncalZElectron+EcalESAlign+HcalCalIterativePhiSym+HcalCalIsoTrkFilter",
+                  "SingleMu"       : "MuAlCalIsolatedMu+MuAlOverlaps+TkAlMuonIsolated+DtCalib+MuAlZMuMu+HcalCalHO",
+                  "SingleMuon"     : "TkAlMuonIsolated+DtCalib+MuAlCalIsolatedMu+MuAlOverlaps+MuAlZMuMu+HcalCalIterativePhiSym",
+                  "SinglePhoton"   : "HcalCalGammaJet",
+                  "ZeroBias"       : "SiStripCalZeroBias+TkAlMinBias+LumiPixelsMinBias+SiStripCalMinBias", 
```

Comment:
@jonuuukas from the title of the PR I would expect that you change only from single to double quotes, but here you are removing '+EcalTrg'. Is that intended?